how-to-docs/kubernetes-token.md

```diff
@@ -1,15 +1,23 @@
 # How to get a Kubernetes token for the UK cluster
```

Comment:
Rename to `kubernetes-user-token.md` to keep similar naming standard with the robots file.